test/test_partitionedgraph.jl

```diff
@@ -0,0 +1,113 @@
+using Test
+using NamedGraphs
+using NamedGraphs:
+  spanning_forest, spanning_tree, forest_cover, PartitionEdge, PartitionVertex, parent
+using Dictionaries
+using Graphs
+
+@testset "Test Partitioned Graph Constructors" begin
+  nx, ny = 10, 10
+  g = named_grid((nx, ny))
+
+  partitions = [[(i, j) for j in 1:ny] for i in 1:nx]
+  pg = PartitionedGraph(g, partitions)
+  @test vertextype(partitioned_graph(pg)) == Int64
+  @test vertextype(unpartitioned_graph(pg)) == vertextype(g)
+  @test is_tree(partitioned_graph(pg))
+  @test nv(pg) == nx * ny
+  @test nv(partitioned_graph(pg)) == nx
+
+  partition_dict = Dictionary([first(partition) for partition in partitions], partitions)
+  pg = PartitionedGraph(g, partition_dict)
+  @test vertextype(partitioned_graph(pg)) == vertextype(g)
+  @test vertextype(unpartitioned_graph(pg)) == vertextype(g)
+  @test is_tree(partitioned_graph(pg))
+  @test nv(pg) == nx * ny
+  @test nv(partitioned_graph(pg)) == nx
+
+  pg = PartitionedGraph([i for i in 1:nx])
+  @test unpartitioned_graph(pg) == partitioned_graph(pg)
+  @test nv(pg) == nx
+  @test nv(partitioned_graph(pg)) == nx
+  @test ne(pg) == 0
+  @test ne(partitioned_graph(pg)) == 0
+end
+
+@testset "Test Partitioned Graph Vertex/Edge Addition and Removal" begin
+  nx, ny = 10, 10
+  g = named_grid((nx, ny))
+
+  partitions = [[(i, j) for j in 1:ny] for i in 1:nx]
+  pg = PartitionedGraph(g, partitions)
+
+  pv = PartitionVertex(5)
+  v_set = vertices(pg, pv)
+  edges_involving_v_set = boundary_edges(g, v_set)
+
+  #Strip the middle column from pg via the partitioned graph vertex, and make a new pg
+  NamedGraphs.rem_vertex!(pg, pv)
```

Comment:
```suggestion
  rem_vertex!(pg, pv)
```
and similar elsewhere.